src/Viewer/services/utils.test.js

```diff
@@ -0,0 +1,82 @@
+import {binarySearchWithTimestamp} from "./utils";
+
+// This file tests the helper functions inside utils.
+
+/**
+ * Using timestampArray as a reference, create an array named logEvents which
+ * every single element in this array has a key named "timestamp".
+ * @param {Array} logEvents
+ * @param {Array} timestampArray Array of integers
+ */
+function _loadLogEvents(logEvents, timestampArray) {
```

Comment:
```suggestion
/**
 * Using timestampArray as a reference, fill logEvents so that each element is
 * an object with the key "timestamp" and the value of the corresponding
 * timestampArray element.
 * @param {Array} timestampArray Array of integers
  * @param {Array} logEvents
 */
function _loadLogEvents(timestampArray, logEvents) {
```